WebdevPeriod3/wwwroot/js/site.js

```diff
@@ -1,9 +1,9 @@
-﻿document.addEventListener("DOMContentLoaded", function (event) {
-    document.getElementById("nav-toggle").addEventListener("click", (e) => {
-        document.getElementById("side-nav").classList.toggle("hidden");
+﻿$(() => {
```

Comment:
Is there a reason we re-added jQuery into the project besides toggling the class of an element?
If it is only used here and its only purpose is toggling an element I would prefer just using vanilla JS.